src/Movie-Card/Movie-Card.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react'
+import { Link } from 'react-router-dom';
 
-const MovieCard = ({ id, title, poster, switchView }) => {
-    return <img key={id} src={poster} id={id} name={title} alt={title} className='movie-card' height='300' width='200' onClick={() => switchView(id)}/>
+const MovieCard = ({ id, title, poster }) => {
+    return <Link to={'/' + id}><img key={id} src={poster} id={id} name={title} alt={title} className='movie-card' height='300' width='200'/></Link>
```

Comment:
The movies need to be Links as far as I can tell in order to go to the Movie view.

Reply:
Do you think navlink might be easier or would it make a difference?

Reply:
This ended up not being an issue with the code you wrote and I continued working on :)